openpype/hosts/maya/api/workfile_template_builder.py

```diff
@@ -43,7 +43,13 @@ def import_template(self, path):
             ))
 
         cmds.sets(name=PLACEHOLDER_SET, empty=True)
-        new_nodes = cmds.file(path, i=True, returnNewNodes=True)
+        new_nodes = cmds.file(
+            path,
+            i=True,
```

Comment:
```suggestion
            import=True,
```
Better to have full argument names, so there is less guess work.

Reply:
I'm afraid this one has no long name.

<img width="1722" alt="Screen Shot 2023-05-03 at 12 00 45" src="https://user-images.githubusercontent.com/68907585/235887088-1a079aef-6487-4b6a-ac98-0d3df205b1bb.png">